src/pages/dataSetsWip/New.spec.tsx

```diff
@@ -0,0 +1,8 @@
+describe.skip('Data sets add', () => {
+    it('should let you add and save data sets', () => {
```

Comment:
I think the decision was not to write tests but indicate where we want tests? Since the PR is mostly reusing existing components, I assume we would have (or would want) some generic tests for ModelTransfer and such.

The thing of note in this PR that I thought it worth testing is the formatting/parsing between dataElements and dataSetElements, so I'm proposing an integration-level test to assert that occurs properly when creating a new data set